internal/api/server.go

```diff
@@ -232,11 +259,71 @@ func (s *Server) ReshareVault(c echo.Context) error {
 		asynq.Retention(10*time.Minute),
 		asynq.Queue(tasks.QUEUE_NAME))
 	if err != nil {
+		s.logger.Errorf("ReshareVault: Failed to enqueue task: %v", err)
 		return fmt.Errorf("fail to enqueue task, err: %w", err)
 	}
+
+	// Notify plugin server asynchronously
+	go func() {
```

Comment:
@SunshineRider , please don't make request to Plugin server from verifier , the front-end ui part should call the plugin server for this